frontend/src/pages/Chats.tsx

```diff
@@ -1,9 +1,83 @@
 import TopAppBar from '../components/TopAppBar';
+import { useEffect, useState } from 'react';
+
+import { StreamChat } from 'stream-chat';
+import {
+  Chat,
+  Channel,
+  Window,
+  ChannelHeader,
+  MessageList,
+  MessageInput,
+  Thread,
+  LoadingIndicator,
+  ChannelList,
+} from 'stream-chat-react';
+
+import 'stream-chat-react/dist/css/index.css';
+import './Chats.css';
+
+const apiKey = process.env.REACT_APP_STREAM_API_KEY
```

Comment:
This key is actually a public key so you can put it in `frontend/.env.local`. But it's best practice to have a separate application for local and for production. E.g. for local, we can enable dev mode but not for production just yet. For our production frontend, you can add environment variables in the Amplify console [here](https://ap-southeast-2.console.aws.amazon.com/amplify/home?region=ap-southeast-2#/d2dflf7eo6e40g/settings/variables)